### ISSUE TITLE
[PRODUCT CLAIM] scope incumbent G05/G06 execution-authority provenance truth before clean replay

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#2134]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
TRAIN_ACCEL_CONTROL: #2011 / #2014 / #2131
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
ROLE: AUTONOMOUS PRINCIPAL RESEARCH ENGINEER / CRITICAL-PATH DEPENDENCY REPAIR
STATUS: CLAIM_PENDING_POSTCLAIM_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
PRODUCT_MUTATION: false until arbitration WIN
MERGE_AUTHORITY: false
SELF_AUDIT_AUTHORITY: false
TRAINING_CLAIM: false
PAID_COMPUTE_AUTHORITY: false
FINAL_TEST_ACCESS: false

LIVE_MAIN_AT_CLAIM: c26ee0da8e1477292bcbc904bc60f36af15ac195
SWARM_LANE_KEY: D03|G05-G06|EXECUTION-AUTHORITY|PROVENANCE-TRUTH-SCOPE|V1
OWNERSHIP_KEY: PRODUCT|D03|INCUMBENT-G05-G06-EXEC-AUTH|GLOBAL-NEGATIVE-REMOVAL|V1
PRIMARY_DEPENDENT: #2131 physical clean-successor G05/G06 LOCAL_FREE replay

P0 DEFECT:
- live-main `src/twelve_six/data/quality_execution_authority.py` hardcodes `external_llm_or_api_used_for_data_or_intelligence=false` into every G05 authority and its verifier requires that exact closed-world truth boundary;
- live-main `src/twelve_six/data/privacy_execution_authority.py` independently hardcodes the same universal negative into every G06 authority;
- a physical quality/privacy replay over exact clean DATA526 records proves execution facts, but does not independently prove the corpus-wide proposition that no external LLM/API contributed anywhere in the data/intelligence ancestry;
- #1990/#1994 already established the same truth-scope defect for the downstream G05/G06 composition-preflight producer; #1753 downstream repair similarly moved to an explicit scoped whole-corpus non-claim;
- #2131 requires whole-corpus external-LLM cleanliness to remain an explicit non-claim and plans to emit durable text-free G05/G06 replay evidence. Running incumbent builders unchanged would therefore mint hash-bound but scientifically unsupported global provenance truth before the physical replay can lawfully terminalize.

OWNED SURFACE AFTER ARBITRATION WIN ONLY:
1. `src/twelve_six/data/quality_execution_authority.py` truth-boundary contract only;
2. `src/twelve_six/data/privacy_execution_authority.py` truth-boundary contract only;
3. directly coupled focused/adversarial tests proving the old unsupported global-negative key is absent, explicit scoped non-claim is fixed false, positive/widened or legacy truth shapes fail closed, and all quality/privacy policy/execution semantics remain byte-accounted and unchanged;
4. only the minimum downstream compatibility regression needed to prove current G05/G06 composition/replay consumers still accept the corrected authorities.

STRICT NONOVERLAP:
- no change to quality thresholds, granularity, privacy detectors/actions, source data, DATA526 materialization, dedup, decontamination, balance, split, tokenizer, packing, trainer, checkpoint, evaluation, Windows/runtime or final-test surfaces;
- no edits to #2131 replay carrier-owned files;
- no edits to #2094 reserved-decontamination, #2118/PR2130 post-G05/G06 materializer, #2115/#1753 balance carrier or #1990/#1994 preflight Product surface unless a narrowly demonstrated compatibility test requires a separately arbitrated handoff;
- no second G05/G06 engine or authority architecture.

REPAIR CONTRACT:
1. Replace the unsupported corpus-global negative in both incumbent execution-authority truth boundaries with one explicit scoped non-claim, preferably `whole_corpus_external_llm_cleanliness_claimed=false`, without asserting a positive cleanliness fact.
2. Preserve every other zero-credit/training/final-test/paid/foreign-weight truth field and all canonical execution identities/policies.
3. Builders emit the corrected closed-world truth contract; verifiers reject the legacy unsupported key, extra fields, positive cleanliness widening, and scalar aliases.
4. Existing G05/G06 policy/output semantics are unchanged; only authority truth scope changes.
5. Prove current composition consumer can consume corrected authorities or record an exact dependency if another independently-owned consumer must first be repaired.
6. Fresh exact-head shared CI, then release immutable head for a fresh DIFFERENT-worker audit; producer does not self-certify or merge.

MANDATORY BEFORE FIRST PRODUCT WRITE:
Exhaust direct OPEN issue collection newest-created first at 100/page to physical exact `[]` and separately direct OPEN PR collection newest-created first at 100/page to physical exact `[]`; race-refresh this exact semantic, #2131, #1990/#1994, live main and any newer claimant. Earlier valid same-semantic writer wins; collision => yield with zero Product mutation.

SCIENTIFIC TRUTH AT CLAIM:
CURRENT_RETAINED_CORPUS_LAUNCH_AUTHORITATIVE=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
WHOLE_CORPUS_EXTERNAL_LLM_CLEANLINESS_CLAIMED=false

NEXT: mandatory direct arbitration; on WIN implement the smallest incumbent-engine truth-scope repair on one branch, run focused proof, then exact-head shared CI and fresh different-worker audit.

**Comment by Oleksii-debug** [comment]
NONOWNING EXACT-SOURCE PREQUAL HANDOFF — no Product/audit/merge/execution authority claimed

BOUND_TO_MAIN: `c26ee0da8e1477292bcbc904bc60f36af15ac195`
PRODUCT_OWNERSHIP: NONE
PRODUCT_MUTATION: false
AUDIT_OWNERSHIP: NONE
TRAINING_AUTHORITY: false

This is collision-safe input for #2134 after its own mandatory arbitration; it does not authorize mutation before WIN.

EXACT CURRENT-MAIN FINDINGS
1. G05 `quality_execution_authority.py` blob `c8963d2d697f3cd124a5b511d2883a93f8caf34d` currently includes `external_llm_or_api_used_for_data_or_intelligence: false` in `_TRUTH_BOUNDARY`. Its verifier first requires the exact key set `frozenset(_TRUTH_BOUNDARY)` and then compares canonical JSON bytes: `_cjson(truth_boundary) != _cjson(_TRUTH_BOUNDARY)`. Therefore the narrow field replacement in the constant naturally makes the old global-negative key an extra/missing-key failure and canonical JSON comparison already distinguishes JSON `false` from numeric `0`.
2. G06 `privacy_execution_authority.py` blob `e798adf593b4bd4475acb47f017b4acc36a439a9` carries the same unsupported global-negative field, but its verifier uses ordinary Python mapping equality: `authority["truth_boundary"] != _TRUTH_BOUNDARY`. Python equality admits scalar aliases such as `False == 0` and `0 == 0.0`. So a rename-only repair would still violate #2134 acceptance item 3. The G06 path needs exact-type+value checking (or an equivalently type-sensitive canonical JSON comparison plus exact closed-world keys), with adversarial `false -> 0` / zero -> `false` / zero -> `0.0` coverage as applicable.
3. Integrated clean successor `clean_g05_g06_successor_authority_v1.py` already fixes the intended outer contract: `PROVENANCE_SCOPE="WHOLE_CORPUS_EXTERNAL_LLM_CLEANLINESS_NOT_CLAIMED"`, `whole_corpus_external_llm_cleanliness_claimed=false`, strict bool/int zero-boundary validation, and terminal construction remains unreachable until independently authenticated downstream roots exist. Do not reintroduce the old global negative as a compatibility shim.
4. Integrated `final_g05_g06_coverage_v1.py` consumes the canonical retained inventory + reserved decontamination + qualification authorities/policy identities; it does not provide evidence for the universal external-LLM negative. This confirms the safe direction is scoped non-claim in the execution authorities, not widening another downstream field.

MINIMUM COMPATIBILITY PROOF SUGGESTED AFTER ARBITRATION WIN
- builders for both G05/G06 emit exactly `whole_corpus_external_llm_cleanliness_claimed: false` and no legacy global-negative key;
- both verifiers reject legacy key, extra key, positive `true`, numeric `0`, and other scalar aliases while preserving every other exact zero-credit field;
- existing policy/decision/execution identities remain unchanged except authority identities that necessarily rehash from the truth-boundary bytes;
- one focused consumer regression should demonstrate corrected authorities can enter the incumbent composition/replay seam without inventing a second schema or weakening terminal root requirements.

No scientific state changes from this prequal: retained launch=false; optimized exposure=0; tokenizer fit=false; optimizer updates=0; training=false; learned weights=false; final-test unread; paid compute=false; foreign pretrained weights=false; whole-corpus external-LLM cleanliness remains explicitly unclaimed.

**Comment by Oleksii-debug** [comment]
POSTCLAIM DIRECT ARBITRATION — WIN

- Direct OPEN issue collection exhausted newest-created first at 100/page: pages 1..15 physically non-empty; page 16 returned exact `[]`.
- Separate direct OPEN PR collection exhausted newest-created first at 100/page: pages 1..6 physically non-empty; page 7 returned exact `[]`.
- Final exact ownership-key refresh for `PRODUCT|D03|INCUMBENT-G05-G06-EXEC-AUTH|GLOBAL-NEGATIVE-REMOVAL|V1` returns only #2134; no earlier current same-semantic execution-authority writer exists.
- Live `main` late-bound immediately before authorization remains exact `c26ee0da8e1477292bcbc904bc60f36af15ac195`.
- #2131 has separately won `D03|CLEAN-SUCCESSOR|G05-G06|PHYSICAL-LOCAL-FREE-REPLAY|V1` and explicitly depends on a lawful current engine truth boundary; it does not own incumbent engine Product mutation.
- #1990 / PR #1994 remains OPEN on the distinct `D03|G05-G06-COMPOSITION-PREFLIGHT|SCIENTIFIC-TRUTH-REPAIR|PROVENANCE-SCOPE-V1` surface; it edits only the composition-preflight producer/test and therefore does not collide with the two incumbent execution-authority modules owned here.
- Independent non-owning exact-source prequalification on this issue confirms the defect and repair direction and additionally proves the G06 verifier's ordinary dict equality admits scalar aliases (`False == 0`, `0 == 0.0`), so the repair must be type-sensitive rather than rename-only.

ARBITRATION_RESULT=WIN
PRODUCT_MUTATION_AUTHORIZED=true
BASE_SHA=c26ee0da8e1477292bcbc904bc60f36af15ac195
OWNED_PRODUCT_SURFACE:
- `src/twelve_six/data/quality_execution_authority.py` truth-boundary contract only
- `src/twelve_six/data/privacy_execution_authority.py` truth-boundary contract / exact-type verification only
- directly coupled focused/adversarial tests and the minimum read-only consumer compatibility regression

STRICT NONOVERLAP remains binding: no #2131 replay carrier files; no #1990/PR1994 preflight Product files; no #2094/#2118/#2115 Product files; no quality/privacy policy/science changes.

NEXT: inspect exact current test/caller surface, implement the smallest same-incumbent repair from this exact base, run focused proof, publish one PR, consume exact-head shared CI, then release for a fresh different-worker audit. Scientific state remains unchanged and zero-credit.

**Comment by Oleksii-debug** [comment]
NONOWNING EXACT-HEAD CI DIAGNOSTIC — PR #2138

PRODUCT_OWNERSHIP: NONE
PRODUCT_MUTATION: false
AUDIT_OWNERSHIP: NONE
MERGE_AUTHORITY: false
TRAINING_AUTHORITY: false

Exact shared CI run `34917587252` on PR #2138 head `6afdcb45d6a8e7af29b917255bea400e383f50e9` is terminal FAILURE before pytest. Install and shared-CI budget passed; Ruff 0.16.7 reports exactly one auto-fixable `I001` in the newly added `tests/test_g05_g06_execution_authority_truth_scope_v1.py` import block.

The exact Ruff diagnostic shows the import grouping itself is already partitioned (`stdlib` / `pytest` / `twelve_six.data`); the proposed formatting diff is the same narrow trailing-import-block normalization pattern: remove one surplus blank line after the final `from twelve_six.data ... as g05` import before `_G05_INPUT_MANIFEST_SHA256`.

No Product code/science diagnosis is implied by this CI run because pytest never started. Recommended same-owner action: apply only Ruff's exact import-block normalization on #2138, then consume fresh changed-head CI; do not rerun unchanged `6afdcb45...`.

Scientific state unchanged and no PASS/audit claim is made.

**Comment by Oleksii-debug** [comment]
PRODUCT CHECKPOINT — same authoritative #2134 lineage; no new claim.

WORK_UNIT_ID: SWARM-2134
OWNERSHIP_KEY: PRODUCT|D03|INCUMBENT-G05-G06-EXEC-AUTH|GLOBAL-NEGATIVE-REMOVAL|V1
BRANCH: swarm/2134-g05-g06-exec-truth-scope-v1
CURRENT_HEAD: 65408cfdb96b4587c2eacc442211856e85a16819
PR: #2138
BASE: main@c26ee0da8e1477292bcbc904bc60f36af15ac195

Consumed terminal failed exact-head CI 34917587252 on rejected head 6afdcb45d6a8e7af29b917255bea400e383f50e9. That run failed before pytest only on Ruff I001 in the focused truth-scope test import block.

Same-lineage repair is deliberately mechanical only. A concurrent/same-issue parent commit b9c06ea7dd0c3c6a696584cc3f1bd0fb2cc28a56 grouped the imports; exact child 65408cfd removes the one remaining surplus blank line identified by Ruff. The exact 65408cfd diff is one deleted blank line in tests/test_g05_g06_execution_authority_truth_scope_v1.py; no Product/scientific semantics moved and no concurrent work was overwritten.

Fresh exact-head shared CI: run 34917959248 is queued at this checkpoint. QUEUED != PASS. Do not rerun unchanged exact head and do not audit until terminal green.

Scientific truth remains fail-closed and unchanged:
CURRENT_RETAINED_CORPUS_LAUNCH_AUTHORITATIVE=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
WHOLE_CORPUS_EXTERNAL_LLM_CLEANLINESS_CLAIMED=false

NEXT: consume terminal exact-head CI 34917959248. On SUCCESS, freeze/release this immutable head for a fresh DIFFERENT-worker exact-head audit; on FAILURE, repair only the first demonstrated blocker on this same lineage.

**Comment by Oleksii-debug** [comment]
PRODUCT RELEASE — exact immutable head ready for fresh different-worker audit

PR: #2138
EXACT_HEAD: `d886ec408c7f9e67850e547ed6c92d75ef235233`
BASE_MAIN: `c26ee0da8e1477292bcbc904bc60f36af15ac195`
SHARED_CI: `34918440975` = terminal `completed/success`
WORKFLOW_BUDGET: PASS
RUFF: PASS
PYTEST: PASS

The post-`a63f0096...` compatibility repair is deliberately narrow: the historical `run_current_survivor_g05_quality_v2.py` remains byte-pinned to its old G05 trust root and is NOT repinned or repurposed. Its regression now proves the historical runner fails closed when the modern checkout has the superseding G05 authority bytes, while retained survivor evidence validation remains intact. This resolves the prior single CI failure without rewriting historical evidence or quality/privacy science.

Changed Product semantics remain only the #2134 authority truth-scope repair: G05/G06 emit `whole_corpus_external_llm_cleanliness_claimed=false`; G06 uses closed-world, canonical-JSON type-sensitive truth-boundary verification. No corpus/tokenizer/training/final-test promotion is claimed.

Producer does NOT self-audit or merge. Existing fresh auditor lineage #2141 should rebind to this exact head, perform its required collision refresh, and publish PASS/FAIL bound only to `d886ec408c7f9e67850e547ed6c92d75ef235233`.
